src/lib.rs

```diff
@@ -198,7 +198,7 @@ cfg_if! {
     } else if #[cfg(target_os = "emscripten")] {
         #[path = "use_file.rs"] mod imp;
     } else if #[cfg(target_os = "freebsd")] {
-        #[path = "freebsd.rs"] mod imp;
+        #[path = "sysctl_arandom.rs"] mod imp;
```

Comment:
Could we call the file something that hints at its BSD nature, maybe `bsd_arandom.rs`?